src/views/splash.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>Hack Cambridge Ternary</title>
+        <link rel="stylesheet" type="text/css" href="{{ asset('styles/hc-2018/splash-bundle.css') }}">
+        <link rel="stylesheet" type="text/css" href="https://assets.hackcambridge.com/fonts/PxGroteskRegular-Regular.css">
+        <link rel="stylesheet" type="text/css" href="https://assets.hackcambridge.com/fonts/PxGroteskScreen-Regular.css">
+        <link rel="stylesheet" type="text/css" href="https://assets.hackcambridge.com/fonts/PxGroteskLight-Regular.css">
+        <link rel="stylesheet" type="text/css" href="https://assets.hackcambridge.com/fonts/PxGroteskBold-Regular.css">
+        <link rel="icon" href="{{ asset('images/hc-2018/favicons/favicon.png') }}" sizes="512x512" type="image/png">
+        <link rel="icon" href="{{ asset('images/hc-2018/favicons/favicon.ico') }}" sizes="16x16 24x24 32x32 48x48 64x64 96x96 128x128 256x256" type="image/vnd.microsoft.icon">
+        <link rel="icon" href="{{ asset('images/hc-2018/favicons/favicon.icns') }}" sizes="16x16 24x24 32x32 48x48 64x64 96x96 128x128 256x256 512x512">
+        <link rel="icon" href="{{ asset('images/hc-2018/favicons/favicon.svg') }}" sizes="any" type="image/svg+xml">
+        <meta name=description content="Get hyped for Hack Cambridge 2018">
+        <meta name=keywords content="hack,Cambridge,ternary,sponsorship,hackathon">
+        <meta name=author content="Hack Cambridge Team">
+        <meta name=viewport content="width=device-width, initial-scale=1.0">
+    </head>
+    <body>
+        <main>
+            <section id="splash-intro">
+                <div class="container width">
+                    <div class="seven unit row">
+                        <div class="six unit h">
+                            <object id="splash-hack-cambridge-logo-object" data="{{ asset('images/hc-2018/logo.svg') }}"></object>
+                        </div>
+                        <div id="splash-intro-content" class="seven unit h row">
```

Comment:
It's not clear to me from the name what `h` means, could we name it better?

Reply:
Yup `h` will become `horizontal`.

Reply:
Looking through the CSS I think `h` and `column` can actually be collapsed

Reply:
Done